collect_app/src/main/java/org/odk/collect/android/injection/config/AppDependencyModule.java

```diff
@@ -185,9 +185,16 @@ UserAgentProvider providesUserAgent() {
 
     @Provides
     @Singleton
-    public OpenRosaHttpInterface provideHttpInterface(MimeTypeMap mimeTypeMap, UserAgentProvider userAgentProvider) {
+    public OpenRosaHttpInterface provideHttpInterface(MimeTypeMap mimeTypeMap, UserAgentProvider userAgentProvider, Application application, VersionInformation versionInformation) {
+        String cacheDir;
+        if (!versionInformation.isRelease() || BuildConfig.DEBUG) {
+            cacheDir = application.getCacheDir().getAbsolutePath();
```

Comment:
We've typically used the project's `.cache` dir instead of Android's own cache dir. That might make sense in some cases, but I think it's useful to keep http caching here so that it's easy to clear if needed and to prevent it getting accidentally cleared or messed with.